readthedocs/builds/models.py

```diff
@@ -1070,7 +1070,7 @@ class VersionAutomationRule(PolymorphicModel, TimeStampedModel):
     )
     action_arg = models.CharField(
         _('Action argument'),
-        help_text=_('Value used for the action to perfom an operation'),
+        help_text=_('Value used for the action to perform an operation'),
```

Comment:
This will require a new migration, you can do this with

```
python manage.py makemigrations --name fix_typo
```

But I can push that change if you don't have your local installation working https://docs.readthedocs.io/en/stable/development/install.html

Reply:
Reverted as well.  I will let you do the `manage.py` thing in a separate PR.  Thx.